src/diagnostics/gravitywave_diagnostics.jl

```diff
@@ -0,0 +1,86 @@
+# This file is included in diagnostics.jl
+
+# Gravity-wave related diagnostics
+
+###
+# Eastward Non-Orographic Gravity-Wave Tendency 
+###
+compute_utendnogw!(out, state, cache, time) = compute_utendnogw!(
+    out,
+    state,
+    cache,
+    time,
+    cache.atmos.non_orographic_gravity_wave,
+)
+compute_utendnogw!(_, _, _, _, non_orographic_gravity_wave::T) where {T} =
+    error_diagnostic_variable("utendnogw", non_orographic_gravity_wave)
+
+function compute_utendnogw!(
+    out,
+    state,
+    cache,
+    time,
+    ::T,
+) where {T <: NonOrographicGravityWave}
+    if isnothing(out)
+        # u_waveforcing is not a field:
+        # return Fields.array2field(
+        #     cache.non_orographic_gravity_wave.u_waveforcing,
+        #     axes(state.c),
+        # )
+        return copy(cache.non_orographic_gravity_wave.uforcing)
+    else
+        out .= cache.non_orographic_gravity_wave.u_waveforcing
+    end
```

Comment:
We can start using the simpler system to write diagnostics
```suggestion
if pkgversion(ClimaDiagnostics) >= v"0.2.13"
    return cache.non_orographic_gravity_wave.u_waveforcing
else
    if isnothing(out)
        # u_waveforcing is not a field:
        # return Fields.array2field(
        #     cache.non_orographic_gravity_wave.u_waveforcing,
        #     axes(state.c),
        # )
        return copy(cache.non_orographic_gravity_wave.uforcing)
    else
        out .= cache.non_orographic_gravity_wave.u_waveforcing
    end
end
```

Reply:
Also, you should return Fields if you want to get a NetCDF output (because we can only interpolate Fields)

Reply:
Actually it is a field (the comments are wrong, we should remove them).

Reply:
> Also, you should return Fields if you want to get a NetCDF output (because we can only interpolate Fields)

But it seems that `uforcing` and `vforcing` are already Fields, should I still explicitly return Field, like this?
```
        # return Fields.array2field(
        #     cache.non_orographic_gravity_wave.u_waveforcing,
        #     axes(state.c),
        # )
```

Reply:
No, you don't need to.

Reply:
> Actually it is a field (the comments are wrong, we should remove them).

Yes, the comment was meant to say: if `uforcing` or `vforcing` is _not_ a field, then we have to explicitly convert them before outputting them. It was something I wrote myself as I was learning.